x/vendorinfo/types/messages_vendor_info_test.go

```diff
@@ -18,13 +18,19 @@ func TestMsgCreateVendorInfo_ValidateBasic(t *testing.T) {
 		{
 			name: "invalid address",
 			msg: MsgCreateVendorInfo{
-				Creator: "invalid_address",
+				Creator:          "invalid_address",
+				VendorID:         int32(testconstants.VendorID1),
+				VendorName:       testconstants.VendorName,
+				CompanyLegalName: testconstants.CompanyLegalName,
 			},
 			err: sdkerrors.ErrInvalidAddress,
 		}, {
 			name: "valid address",
 			msg: MsgCreateVendorInfo{
-				Creator: sample.AccAddress(),
+				Creator:          sample.AccAddress(),
+				VendorID:         int32(testconstants.VendorID1),
```

Comment:
Makes sense to add more tests here to check validation of messages as defined in https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/issue-99-cosmos-upgrade-v0.44/proto/vendorinfo/tx.proto:
- min/max value
- check that VID is unit16 ([0, 65535])

Reply:
done